src/native/eventpipe/ep-buffer-manager.c

```diff
@@ -248,6 +248,44 @@ ep_buffer_list_get_and_remove_head (EventPipeBufferList *buffer_list)
 	return ret_buffer;
 }
 
+bool
+ep_buffer_manager_try_reserve_buffer(
```

Comment:
Make these static functions only declared in the .c file (equivalent of a private method on the class in the shared library's pattern).